common/data.re

```diff
@@ -703,62 +694,41 @@ module Timetable = {
     {task: Talk(Speaker.jaredForsyth), duration: Some(45)},
     {task: OpenEnd("After Party at Cafe Leopold"), duration: None},
   ];
-  let calcStep = (pre: entry, step: step): entry => {
-    let fromTime = addDuration(pre.fromTime, pre.duration);
+  let day3Steps = [
+    {
+      task: Misc("Doors open, Registration & Editor Setup"),
+      duration: Some(60),
+    },
+    {task: OpenEnd("Enjoy yourself"), duration: None},
+  ];
+  let calcStep = (fromTime: Js.Date.t, step: step): entry => {
     let toTime =
       switch (step.duration) {
       | None => None
       | Some(_) => Some(addDuration(fromTime, step.duration))
       };
     {task: step.task, duration: step.duration, fromTime, toTime};
   };
-  let rec calcTimetable = (pre: entry, steps: list(step)): list(entry) =>
+  let rec calcTimetable =
+          (fromTime: Js.Date.t, steps: list(step)): list(entry) =>
     switch (steps) {
     | [s, ...rest] =>
-      let next = calcStep(pre, s);
-      [next, ...calcTimetable(next, rest)];
+      let next = calcStep(fromTime, s);
+      [
+        next,
+        ...calcTimetable(addDuration(next.fromTime, next.duration), rest),
+      ];
     | [] => []
     };
-  /* This is just for debugging a table,.. also a good inspiration
-     on how to interpret table data */
-  let logTimetable = (table: list(entry)): unit =>
-    List.iter(
-      (r: entry) => {
-        let task =
-          switch (r.task) {
-          | Talk(speaker) => speaker.name
-          | Misc(msg) => msg
-          | Break(msg) => msg
-          | Workshop(speakers) =>
-            speakers
-            |> List.map((speaker: Speaker.t) => speaker.name)
-            |> String.concat(" ")
-          | OpenEnd(msg) => msg
-          };
-        let fromTime = DateFns.format("DD.MM, HH:mm", r.fromTime);
-        let duration =
-          switch (r.duration) {
-          | Some(d) => {j| ($d min)|j}
-          | None => ""
-          };
-        let timeRange =
-          switch (r.toTime) {
-          | Some(time) =>
-            let toTime = DateFns.format("HH:mm", time);
-            {j|$fromTime - $toTime$duration|j};
-          | None => fromTime
-          };
-        ();
-      },
-      table,
-    );
-  let day1Timetable = [
-    day1StartEntry,
-    ...calcTimetable(day2StartEntry, day1Steps),
-  ];
-  let day2Timetable = [
-    day2StartEntry,
-    ...calcTimetable(day2StartEntry, day2Steps),
+
+  /* TODO: Why is it saying 11am +2 */
+  let startDate = Js.Date.fromString("2019-04-11T11:00:00.000+02:00");
```

Comment:
@okonet the +2 are our timezone CEST. Does that make sense? Can we remove the TODO?